Gopkg.toml

```diff
@@ -23,6 +23,14 @@ required = [
   name = "k8s.io/apimachinery"
   version = "kubernetes-1.13.4"
 
+[[override]]
+  name = "k8s.io/apiserver"
+  version = "kubernetes-1.13.4"
+
+[[override]]
```

Comment:
Why is this added? I don't see it used anywhere.

Reply:
Because of dep. It's a transitive dependency. After I changed the gardener dependency to a newer commit, dep downloaded `master` instead of `kubernetes-1.13.4`, which didn't build. So I had to pin it to its correct version here.